File: ProAgil.API/model/Evento.cs
using System;

namespace ProAgil.API.model
{
    public class Evento
    {
        public string id { get; set; } = Guid.NewGuid().ToString();
        public string Local { get; set; }
        public string DataEvento { get; set; }
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string Lote { get; set; }
    }
}

File: PraAgil-App/src/app/_models/evento.ts
import { Lote } from './Lote';
import { Palestrante } from './Palestrante';
import { RedeSocial } from './RedeSocial';

// tslint:disable-next-line: no-empty-interface
export interface Evento {
    id: number;
    local: string;
    dataEvento: Date;
    tema: string;
    qtdPessoas: number;
    imagemUrl: string;
    telefone: string;
    email: string;
    lotes: Lote[];
    sociais: RedeSocial[];
    eventos: Palestrante[];
}
